Special handling for empty lines in section with language shebang (#413)
MarketSquare/robotframework-tidy: docs/source/transformers/NormalizeNewLines.rst

            Mid Test
                My Step 1    args    args 2    args 3

Language header
----------------

> Files that begins with language marker are handled differently.

Reviewer: ```suggestion
Files that begin with the language marker are handled differently.
```

> If the section contains only language marker and no more than ``section_lines``, it will be not transformed.

Reviewer: ```suggestion
If the section contains only language marker and no more empty lines than the value of ``section_lines``,
it will be not transformed.
```

> Following examples will not be formatted since number of empty lines is lower or equal to default ``section_lines``:

Reviewer: ```suggestion
The following examples will not be formatted since the number of empty lines is lower or equal to default value of ``section_lines``:
```
